livingdoc-converters/src/test/kotlin/org/livingdoc/converters/collection/ListConverterTest.kt

```diff
@@ -0,0 +1,41 @@
+package org.livingdoc.converters.collection
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.livingdoc.api.conversion.Converter
+import org.livingdoc.converters.DefaultTypeConverterContract
+
+internal class ListConverterTest : CollectionConverterContract<List<Any>>(), DefaultTypeConverterContract {
+
+    override val cut = ListConverter()
+    override val collectionClass = List::class.java
+
+    @Test
+    fun `canConvertBoolean`() {
+        val input = "true, false, false, true"
+        val expected = listOf(true, false, false, true)
+
+        assertThat(cut.convert(input, getParameterTypeConverter(listFake::class, "boolean"), null)).isEqualTo(expected)
+    }
+
+    @Test
+    fun `canConvertInt`() {
+        val input = "1, 2, 3, 4"
+        val expected = listOf(1, 2, 3, 4)
+
+        assertThat(cut.convert(input, getParameterTypeConverter(listFake::class, "integer"), null)).isEqualTo(expected)
```

Comment:
these assertions are a little mit long, maybe extract some of the input into variables? like the result of `getParameterTypeConverter(...)` and the result of the `cut.convert(...)` invocation?